routes/mediahub_pornstars.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Why are you creating a new routes file. These should be in mediahub group in the web routes file along with rest.